app/scripts/views/mixins/flow-begin-mixin.js

```diff
@@ -10,10 +10,7 @@ define(function (require, exports, module) {
 
   module.exports = {
     afterRender () {
-      const flowId = this.flow.get('flowId');
-      const flowBegin = this.flow.get('flowBegin');
-
-      this.metrics.logFlowBegin(flowId, flowBegin);
+      this.logFlowEventOnce('begin');
```

Comment:
The `flow.begin` event doesn't interpolate the view name by design, because it makes the funnel queries easier (and faster). It's possible some other flow events in the future may not interpolate either. Anyway, for that reason, `viewName` is optional.